src/Twig/Extension/ReleaseNotesExtension.php

```diff
@@ -28,16 +28,19 @@
 
 class ReleaseNotesExtension extends AbstractExtension
 {
+    /**
+     * @return \Twig\TwigFilter[]
+     */
     public function getFilters()
     {
         return [
             // format filters
-            new TwigFilter('removeWikiLink', [$this, 'removeWikiLink']),
+            new TwigFilter('removeWikiLink', fn (string $data): string => $this->removeWikiLink($data)),
```

Comment:
This is why we need partial application for PHP 8.1. :-)  I somewhat disagree here.  Yes, it's a bit more refactorable, but it's also an extra function stack call and a bit longer.  I... suppose I could go either way here, but it's unnecessary.

Reply:
Would love to see method pointers for these cases which does not longer need reflection to resolve the method from the string....